libs/media-player/src/lib/audio-clip-player.tsx

```diff
@@ -0,0 +1,45 @@
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import { Box } from '@mui/material';
+import { useState } from 'react';
+
+interface IPlayButtonProps {
+    isDisabled: boolean;
+    onButtonClick: () => void;
+}
+
+export interface IPlayButton {
+    (props: IPlayButtonProps): JSX.Element;
+}
+
+export interface AudioClipPlayerProps {
+    audioUrl: string;
+    PlayButton?: IPlayButton;
+}
+
+export function AudioClipPlayer({ audioUrl, PlayButton }: AudioClipPlayerProps) {
+    const [canPlay, setCanPlay] = useState(false);
+
+    const handleMediaPlayerIconClick = () => {
+        new Audio(audioUrl).play().catch(console.log);
```

Comment:
We should probably have an error message and disable the button whenever the play attempt fails and until the audio URL is changed.

Reply:
done